Flat/src/ModulesBTagSFs.cc

```diff
@@ -111,8 +111,7 @@ void PandaAnalyzer::JetBtagSFs()
       for (unsigned int iJ=0; iJ!=nJ; ++iJ) {
         panda::Jet *jet = bCandJets.at(iJ);
         bool isIsoJet=false;
-        if (!analysis->fatjet || // if we do not consider fatjets, everything is an isojet 
-            std::find(isoJets.begin(), isoJets.end(), jet) != isoJets.end()) // otherwise, explicitly check isojet
+        if (std::find(isoJets.begin(), isoJets.end(), jet) != isoJets.end())
```

Comment:
Why this change?

Reply:
This is weird, I haven't touched (not even opened) that file over last days.

Reply:
It was in this commit, 6 days ago: https://github.com/sidnarayanan/PandaAnalysis/pull/59/commits/f61b801eaa2c6c5e03c7e55a13a402112567072a

Reply:
Weird, it wasn't intentional